public/connect4.css

```diff
@@ -0,0 +1,261 @@
+@font-face {
+  font-family: "HK Grotesk";
+  src:
+    url("fonts/HKGrotesk-Regular.woff2") format("woff2"),
+    url("fonts/HKGrotesk-Regular.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  font-family: "HK Grotesk", sans-serif;
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
+  margin: 0;
+}
```

Comment:
Importing this CSS globally has broken the Admin design (probably the `display: flex`).

![image](https://github.com/user-attachments/assets/3ee770bc-e890-4912-9184-2f0553756664)

Can you scope the CSS to the GameGrid component?